ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -146,27 +145,11 @@ void ggml_backend_sycl_print_sycl_devices() {
     }
 }
 
-static inline int get_sycl_env(const char *env_name, int default_val) {
```

Comment:
If we are removing this, is there any other addition where sycl env is called for logging?

Reply:
Same comment.
I suggest keeping this function.

Reply:
According to PR #9709 , the backend specific logging system was supposed to be replaced with a common logging system. Although I agree that this will enable all debug logs if `--log-verbose` cmdline argument is passed or enabled by default in test-backend-ops.

cc @slaren I think best here is to enable them only when something like `GGML_BACKEND_DEBUG=1` is enabled.

Reply:
Sounds good. for platform benchmark purposes it makes sense to have the platform specific debug on.